lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -106,17 +106,6 @@ void GUIUtils::displayBottomLine(String msg)
 #endif 
 }
 
-void GUIUtils::displayEndLine(String msg)
```

Comment:
Wich is the reason for delete this method? it is deprecated or replace for other?

Reply:
Is replaced in other method